smoketest/k8s/smoketest.bash

```diff
@@ -18,13 +18,11 @@ while [ "$#" -ne 0 ]; do
             cleanKind
             kind create cluster
             kind load docker-image "quay.io/${IMAGE_REPOSITORY}/cryostat3:dev"
-            kind load docker-image "quay.io/${IMAGE_REPOSITORY}/cryostat3-db:dev" 
```

Comment:
do we need to load `quay.io/${IMAGE_REPOSITORY}/cryostat-db:latest`

Reply:
I think so, if the Idea it to pull the image on its own?

Reply:
What that does is tell `kind` to explicitly load the image from your local registry (ex. `podman`) into the cluster. This is used for the case where the image is expected to be in the local registry but *not* on the remote registry (`quay.io`). That's why it's still there for the `cryostat3:dev` image since those are not (yet) pushed to `quay.io`. The new `cryostat-db` image does get pushed to quay, so there is no need to explicitly load that image first - you can just use the k8s YAML Deployment file for the database image, and k8s in kind will go ahead and pull that image if it needs to.